types/entropy__winston-elasticsearch-apm/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
```

Comment:
Hopefully you shouldn't need this lint exception at all anymore? Right?

Reply:
@weswigham yes, you are right. It's fixed.